api/src/main/java/org/pmiops/workbench/google/CloudBillingClientImpl.java

```diff
@@ -23,13 +25,32 @@ public CloudBillingClientImpl(
     this.retryHandler = retryHandler;
   }
 
-  public BillingAccount getBillingAccount(String billingAccountName) throws IOException {
+  @Override
+  public ProjectBillingInfo pollUntilBillingAccountLinked(
```

Comment:
Can you add some comments - what are we actually waiting for here? Is it an issue with GCP consistency after we setBillingAccount? Or is it some background process in Terra that creates the account linkage? What is the expected wait time for this?

Reply:
Added comments in the interface class. 
For more context: https://broadworkbench.atlassian.net/browse/CA-1535